feature/home/src/main/java/com/epmedu/animeal/home/presentation/HomeScreenUI.kt

```diff
@@ -215,25 +217,36 @@ private fun OnState(
                 onScreenEvent(FeedingGalleryEvent.CloseDeletePhotoDialog)
             }
         )
+        state.showMotivateUseGpsDialog -> MotivateUseGpsDialog(
+            onConfirm = { onScreenEvent(MotivateUseGpsEvent.AskUseGps) },
+            onDismiss = {
+                onScreenEvent(MotivateUseGpsEvent.DeclineUseGps)
+                onScreenEvent(FeedingEvent.StartWithoutRouting)
+            }
+        )
         else -> Unit
     }
 }
 
 @Composable
 private fun WillFeedConfirmationDialog(
-    willFeedState: WillFeedState,
+    state: HomeState,
```

Comment:
why pass the whole state instead of just the specific associated state?

Reply:
Will change it